<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainContactsPanel</class>
 <widget class="QWidget" name="MainContactsPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>578</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="AvatarWidget" name="img_avatar">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>80</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>80</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="TitleLabel" name="lb_nickname">
         <property name="text">
          <string>Title label</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="SubtitleLabel" name="lb_username">
         <property name="styleSheet">
          <string notr="true">color: rgb(121, 121, 121);</string>
         </property>
         <property name="text">
          <string>Subtitle label</string>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::LinksAccessibleByMouse</set>
         </property>
         <property name="pixelFontSize" stdset="0">
          <number>15</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="SearchLineEdit" name="le_search"/>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="page_nowChatting">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="ListWidget" name="lv_nowChatting"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_friends">
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="ListWidget" name="lv_friends"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_groups">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="ListWidget" name="lv_groups"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="SegmentedWidget" name="navigation">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>33</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SubtitleLabel</class>
   <extends>QLabel</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>TitleLabel</class>
   <extends>QLabel</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>ImageLabel</class>
   <extends>QLabel</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>AvatarWidget</class>
   <extends>ImageLabel</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>Pivot</class>
   <extends>QWidget</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>SegmentedWidget</class>
   <extends>Pivot</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>LineEdit</class>
   <extends>QLineEdit</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>SearchLineEdit</class>
   <extends>LineEdit</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>ListWidget</class>
   <extends>QListWidget</extends>
   <header>qfluentwidgets</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resource/resources.qrc"/>
 </resources>
 <connections/>
</ui>
